testssl.sh

```diff
@@ -13,7 +13,7 @@
 # Project lead and initiator: Dirk Wetter, copyleft: 2007-today, contributions so far see CREDITS.md
 # Main contributions from David Cooper
 #
-# License: GPLv2, see http://www.fsf.org/licensing/licenses/info/GPLv2.html
+# License: GPLv2, see https://spdx.org/licenses/GPL-2.0-only.html
```

Comment:
maybe that would make sense -- can't tell whether I want the LF here -- but https://www.gnu.org/licenses/old-licenses/gpl-2.0.html would work too.

Reply:
I have seen that this is used currently in many FOSS projects for license reference. I guess either will work.